app/routes/calendar/$eventId.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useParams } from '@remix-run/react';
+import EventModal from '~/components/common/eventModal';
+import { useStore } from '~/store/useStore';
+import { CalEvent } from '~/utils/interfaces';
+
+const EventDetails = () => {
+  const { events: eventsList } = useStore((state) => state);
+  const [calendarEvent, setCalendarEvent] = React.useState<any>({
+    title: '',
+    start: undefined,
+    end: undefined,
+    location: '',
+    description: '',
+    attendees: [],
```

Comment:
can we use this as a variable and pass here instead of the direct obj?

Reply:
removed it